### PR TITLE
Transfers in Event Store and CasperClient

### PR DESCRIPTION
**Pull request by zie1ony**
### Overview

- Collect transfer events from event stream into event store.
- Serve transfers via http endpoint `/transfers` by purseUref.
- Add it to the CasperClient.

Example of CasperClient use:
https://github.com/casper-ecosystem/casper-client-sdk-example/blob/main/src/transferInfo.js

### Which JIRA ticket does this PR relate to?
https://casperlabs.atlassian.net/browse/ECO-784
https://casperlabs.atlassian.net/browse/ECO-785

### CODE REVIEW

File: packages/event_store/src/storage.js [hunk 1]
```diff
@@ -114,6 +143,22 @@ class Storage {
             }
         });
     }
+
+    async findTransfers(purseUref) {
+        return this.models.Transfer.findAll({
+            where: {
+                [Op.or]: [
+                    {
+                        sourcePurse: purseUref
```
Comment (AbnerZheng) [inline packages/event_store/src/storage.js:152]: There is no index for sourcePurse and targePurse, the find-all method could be very slow.

Comment (zie1ony) [reply packages/event_store/src/storage.js:152]: done

**Review by AbnerZheng — APPROVED** [review]
LGTM. thank you for update.

**Pull request merged**